apps/web/src/app/api/events/[id]/cancel/route.ts

```
@@ -0,0 +1,32 @@
+import { NextResponse } from 'next/server';
+import { CancellationQueue } from '../../../../../../../../services/payments/cancellationQueue';
```

ESLint

[error] Replace `'../../../../../../../../services/payments/cancellationQueue'` with `"../../../../../../../../services/payments/cancellationQueue"`

(prettier/prettier)